looprpc/client.proto

```diff
@@ -854,6 +854,11 @@ message LiquidityParameters {
     The confirmation target for loop in on-chain htlcs.
     */
     int32 htlc_conf_target = 17;
+
+    /*
+    The address to use for autoloop loop out HTLCs.
```

Comment:
nit: do you mean for the the destination address?

Reply:
Yes, the address where the funds are going to after we sweep the HTLC